image/cli/mascli/functions/pipeline_config

```diff
@@ -228,9 +228,22 @@ function pipeline_config() {
     SLS_ENTITLEMENT_FILE="/workspace/entitlement/$(basename $SLS_LICENSE_FILE_LOCAL)"
   fi
 
-  echo
-  echo_h2 "Configure UDS"
-  prompt_for_input "UDS Contact Email" UDS_CONTACT_EMAIL
-  prompt_for_input "UDS Contact First Name" UDS_CONTACT_FIRSTNAME
-  prompt_for_input "UDS Contact Last Name" UDS_CONTACT_LASTNAME
+  # Check for DRO support in MAS
```

Comment:
User doesn't care.  No-one should have to decide between DRO or UDS during the interactive install, something gets set up and they don't care what.

`MAS_CHANNEL` will never be any of these values, it's in the format 8.10.x, 8.11.x, etc

Reply:
Updated to use MAS_CHANNEL in catalog_utils